homeassistant/components/philips_js/media_player.py

```diff
@@ -19,10 +19,12 @@
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.trigger import PluggableAction
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
 from . import LOGGER as _LOGGER, PhilipsTVDataUpdateCoordinator
 from .const import DOMAIN
+from .device_trigger import async_get_turn_on_trigger
```

Comment:
We shouldn't import a name from another platform. I suggest we move the name to a non platform similar as webostv.